performance/shared/src/main/kotlin/org/jetbrains/benchmarksLauncher/launcher.kt

```diff
@@ -46,11 +46,23 @@ abstract class Launcher {
         }
     }
 
+    enum class LogLevel { DEBUG, OFF }
+
+    class Logger(val level: LogLevel = LogLevel.OFF) {
+         fun log(message: String, messageLevel: LogLevel = LogLevel.DEBUG) {
+            if (messageLevel == level) {
+                printlnStderr("[$level] $message")
```

Comment:
Maybe also print current time?

Reply:
And maybe print single '.' every attempt - because `cinterop/stringToKotlin` takes about minute, so single line in a hour is not very informative.